packages/binding-http/src/http-server.ts

```diff
@@ -769,21 +770,21 @@ export default class HttpServer implements ProtocolServer {
                                 } else {
                                     thing
                                         .readProperty(segments[3], options)
-                                        // property.read(options)
-                                        .then((value: any) => {
+                                        .then(async (value: InteractionOutput) => {
                                             const contentType = ProtocolHelpers.getPropertyContentType(
                                                 thing.getThingDescription(),
                                                 segments[3],
                                                 "http"
                                             );
-                                            const content = ContentSerdes.get().valueToContent(
+                                            /*const content = ContentSerdes.get().valueToContent(
                                                 value,
                                                 <any>property,
                                                 contentType
-                                            );
-                                            res.setHeader("Content-Type", content.type);
+                                            );*/
+                                            res.setHeader("Content-Type", contentType); // content.type);
                                             res.writeHead(200);
-                                            content.body.pipe(res);
+                                            res.end(await value.value() + "");
+                                            // content.body.pipe(res);
```

Comment:
This commented code was needed to support Streams. This means that we'll use support for ExposedThing to answer with a stream of data, it can only answer with DataSchema values.

Reply:
added again